### PR TITLE
BrenKt-SK897-TRANS-WEB-Email-Exchange-adding-domain

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.es-es.md [hunk 1]
```diff
@@ -5,112 +5,151 @@ excerpt: 'Cómo añadir un dominio a un servicio Exchange'
...
+- **Si ha introducido un dominio no gestionado por OVHcloud**: el modo no autoritario se configurará por defecto.
 
-- **Si ha seleccionado en la lista un dominio gestionado por OVH**, deberá elegir uno de los dos modos que se describen a continuación.
+- **Si ha seleccionado en la lista un dominio gestionado por OVHcloud**: deberá elegir entre dos modos.
 
 |Modo|Descripción|
 |---|---|
-|Autoritario|Es el modo adecuado cuando solo utiliza con el dominio la solución Exchange. No permite utilizar otra solución de correo conjuntamente con Exchange.|
-|No autoritario|Es el modo adecuado cuando utiliza con el dominio la solución Exchange conjuntamente con otra solución de correo. Debe indicar el servidor de su otra solución de correo.|
+|Autónomo|Es conveniente si solo utiliza su solución Exchange o Email Pro con su nombre de dominio. No permite utilizar otra solución de correo conjuntamente con el servicio.|
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.es-es.md]: @BrenKt , are you sure this should be the right translation ? (compared to previous version). Please check in the Control Panel

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.es-es.md [hunk 2]
```diff
@@ -5,112 +5,151 @@ excerpt: 'Cómo añadir un dominio a un servicio Exchange'
...
+Una vez que haya añadido los dominios, ya puede configurar sus cuentas de correo con ellos. Para ello, acceda a la pestaña `Cuentas de correo`{.action} de su Exchange. Si lo necesita, puede contratar cuentas adicionales utilizando el botón `Acción`{.action}/`Contratar cuentas`{.action} o `Añadir cuenta`{.action}.
 
-### 4. Configurar y utilizar las cuentas
+Le recordamos que todas las direcciones creadas en su servicio podrán ver en el directorio el conjunto de direcciones de este servicio, incluidas aquellas que posean un nombre de dominio diferente.
 
-Una vez que haya añadido los dominios, ya puede configurar sus cuentas Exchange con dichos dominios. Para ello, abra la pestaña `Cuentas de correo`{.action} de su servicio Exchange. Desde ese mismo lugar podrá también contratar cuentas adicionales con el botón `Añadir cuentas de correo`{.action} o `Añadir una cuenta`{.action}.
+Una vez que haya configurado las cuentas, ¡ya puede utilizarlas! Para ello, OVHcloud pone a su disposición el **webmail** disponible en la dirección <https://www.ovh.com/fr/mail/>. Para un uso óptimo de su dirección en un programa, asegúrese de que sea compatible con el servicio.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.es-es.md]: FR link

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md [hunk 3]
```diff
@@ -1,117 +1,155 @@
 ---
-title: 'Aggiungere un dominio su Exchange'
-slug: aggiungere-dominio-su-exchange
-excerpt: 'Come aggiungere un dominio al servizio di posta Exchange'
-section: 'Configurazione dell’offerta Exchange'
-order: 4
+title: 'Dodanie domeny do usługi Exchange'
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md]: Error here, the whole frontmatter has been switched from IT to PL

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md [hunk 4]
```diff
@@ -1,117 +1,155 @@
...
+**Questa guida ti mostra come aggiungere un dominio alla piattaforma Exchange o Email Pro.**
 
 ## Prerequisiti
 
-- Disporre di una [soluzione Exchange](https://www.ovhcloud.com/it/emails/){.external}
+- Disporre di una soluzione [Exchange](https://www.ovhcloud.com/it/emails/) o [Email Pro](https://www.ovhcloud.com/it/emails/email-pro/))
 - Aver registrato uno o più domini
-- Poter modificare la configurazione del dominio (zona DNS)
-- Avere accesso allo [Spazio Cliente OVH](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it){.external}
+- Poter modificare la configurazione del dominio ([zona DNS](https://docs.ovh.com/it/domains/web_hosting_modifica_la_tua_zona_dns/))
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md]: please check translation and get rid of double ()

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md [hunk 5]
```diff
@@ -1,117 +1,155 @@
...
 
-> [!primary]
->
-> Nello Spazio Cliente OVH il nome dei servizi Exchange inizia con **hosted-** o **private-**, seguito da una parte del tuo identificativo cliente e una cifra (1 per il primo servizio installato, 2 per il secondo, ecc...).
->
+- **Exchange**: Clicca su `Microsoft`{.action} e poi su `Exchange`{.action}. 
+- **Email Pro**: Clicca su `Email Pro`{.action}.
 
-### Step 2: aggiungi un dominio
+e seleziona il nome del servizio.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md:39]: some part is missing here

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md [hunk 6]
```diff
@@ -1,117 +1,155 @@
...
+> Tutti gli indirizzi creati sul tuo servizio di posta elettronica saranno in grado di visualizzare nella stessa directory tutti gli indirizzi email associati al servizio, inclusi quelli con un dominio diverso. Se vuoi che alcuni domini non vengano visualizzati nella stessa directory, è necessario ordinare una nuova [soluzione Exchange o Email Pro](https://www.ovhcloud.com/it/emails/) per il dominio o i domini interessati.
 >
 
-![Exchange](images/add_domain_exchange_step1.png){.thumbnail}
+![Add Domain](images/add_domain_exchange_step1.png){.thumbnail}
 
 Scegli una delle opzioni disponibili:
 
-- **Seleziona un dominio dalla lista**: vengono mostrati soltanto i domini che utilizzano la configurazione OVH e inseriti come contatti nel tuo identificativo cliente;
+- **seleziona un dominio dalla lista**: vengono mostrati solo i domini configurati con OVHcloud e associati al tuo identificativo cliente
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md]:
```suggestion
- **Seleziona un dominio dalla lista**: vengono mostrati solo i domini configurati con OVHcloud e associati al tuo identificativo cliente
```

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md [hunk 7]
```diff
@@ -1,117 +1,155 @@
...
+>
+> Prima di eliminare account email, assicurati che non siano utilizzati. Potrebbe essere necessario eseguire un backup di questi account. Se necessario, consulta la guida [Migrare manualmente il tuo indirizzo email](https://docs.ovh.com/it/emails/migrare-i-indirizzi-email-manualmente/) che ti descriverà come esportare i dati di un account dal tuo Spazio Cliente OVH o da un client di posta.
+
+Clicca sulla scheda `Domini associati`{.action} della tua piattaforma. Dalla tabella, la colonna `Account` ti indica il numero di account associati ai domini della tua lista.
+
+![emailpro](images/add_domain_exchange_step6.png){.thumbnail}
+
+Se al dominio da scollegare sono associati account email, è possibile:
+
+- **Associa gli account a un altro dominio**: accedi alla scheda `Account email`{.action}. A destra degli account da modificare, clicca sul pulsante `...`{.action} e poi `su Modifica`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md]:
```suggestion
- **Associa gli account a un altro dominio**: accedi alla scheda `Account email`{.action}. A destra degli account da modificare, clicca sul pulsante `...`{.action} e poi su `Modifica`{.action}.
```

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pl-pl.md [hunk 8]
```diff
@@ -1,117 +1,155 @@
...
+**Dowiedz się, jak dodać domenę do platformy Exchange lub E-mail Pro.**
 
 ## Wymagania początkowe
 
-- Wykupienie [usługi Exchange](https://www.ovhcloud.com/pl/emails/hosted-exchange/){.external}.
+- Posiadanie rozwiązania [Exchange](https://www.ovhcloud.com/pl/emails/) lub [Email Pro](https://www.ovhcloud.com/pl/emails/email-pro/))
 - Zarejestrowana domena lub kilka domen.
-- Możliwość modyfikacji konfiguracji Twojej domeny (jej strefy DNS).
-- Dostęp do [Panelu klienta OVH](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl){.external}.
+- Możliwość modyfikacji konfiguracji Twojej domeny ([strefy DNS](https://docs.ovh.com/pl/domains/hosting_www_jak_edytowac_strefe_dns/))
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pl-pl.md]: Same remark as for IT

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pl-pl.md [hunk 9]
```diff
@@ -1,117 +1,155 @@
...
+![emailpro](images/add_domain_exchange_step6.png){.thumbnail}
+
+Jeśli konta e-mail są przypisane do domeny, którą chcesz odłączyć, masz 2 możliwości:
+
+- **Przypisz konta do innej domeny**: przejdź do zakładki `Konta e-mail`{.action}. Po prawej stronie kont do zmiany kliknij przycisk `...`{.action}, a następnie `Zmień`{.action}.
+    ![emailpro](images/add_domain_exchange_step8.png){.thumbnail}
+    W oknie modyfikacji możesz zmienić nazwę domeny przypisanej do konta w rozwijanym menu.
+    ![emailpro](images/add_domain_exchange_step9.png){.thumbnail}
+
+- **Usuń konta z Twojej platformy**: przejdź do zakładki `Konta e-mail`{.action}. Po prawej stronie konta, które chcesz usunąć, kliknij przycisk `...`{.action}, a następnie `Zresetuj to konto`{.action} lub `Zresetuj`{.action}
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pl-pl.md]:
```suggestion
- **Usuń konta z Twojej platformy**: przejdź do zakładki `Konta e-mail`{.action}. Po prawej stronie konta, które chcesz usunąć, kliknij przycisk `...`{.action}, a następnie `Zresetuj to konto`{.action} lub `Zresetuj`{.action}.
```

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pt-pt.md [hunk 10]
```diff
@@ -0,0 +1,155 @@
+---
+title: Adicionar um domínio a uma plataforma de e-mail
+slug: adicionar-dominio-exchange
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pt-pt.md]: please use EN slug, same as en-gb

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pt-pt.md [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+
+Adicionar um domínio a um serviço Exchange é indispensável para poder utilizar as suas contas incluídas neste último. Também é possível adicionar vários domínios a um serviço Exchange ou E-mail Pro.
+
+**Saiba como adicionar um domínio à plataforma Exchange ou E-mail Pro.**
+
+## Requisitos
+
+- Dispor de uma solução [Exchange](https://www.ovhcloud.com/pt/emails/) ou [Email Pro](https://www.ovhcloud.com/pt/emails/email-pro/)).
+- Dispor de um ou vários domínios.
+- Ter a possibilidade de alterar a configuração do domínio ([zona DNS](https://docs.ovh.com/pt/domains/alojamento_partilhado_como_editar_a_minha_zona_dns/)).
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pt-pt.md]: same remark as for IT and PL

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pt-pt.md [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+
+A janela apresenta agora informações sobre a configuração dos modos.
+
+- **Se indicou um domínio não gerido pela OVHcloud**: o modo não autoritário será configurado por predefinição.
+
+- **Se selecionou na lista um domínio gerido pela OVHcloud**: deverá escolher entre dois modos.
+
+|Modo|Descrição|
+|---|---|
+|Authoritative|Para usar o seu domínio apenas com o Exchange ou o E-mail Pro. Não permite a utilização de outra solução de e-mail com o seu serviço.|
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pt-pt.md]: it is in EN in the PT Control Panel ?

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pt-pt.md [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+![emailpro](images/add_domain_exchange_step6.png){.thumbnail}
+
+Se existirem contas de e-mail associadas ao nome de domínio que pretende desassociar, tem 2 possibilidades:
+
+- **Associar as contas a outro domínio**: aceda ao separador `Contas de e-mail`{.action}. À direita das contas a modificar, clique no botão `...`{.action} e depois `Alterar`{.action}.
+    ![emailpro](images/add_domain_exchange_step8.png){.thumbnail}
+    Na janela de alteração, pode alterar o domínio associado à conta através do menu pendente.
+    ![emailpro](images/add_domain_exchange_step9.png){.thumbnail}
+
+- **Eliminar as contas da sua plataforma**: aceda ao separador `Contas de e-mail`{.action}. À direita da conta a eliminar, clique no botão `...`{.action} e, a seguir, `Renovar esta conta`{.action} ou `Reinicializar`{.action}
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.pt-pt.md]:
```suggestion
- **Eliminar as contas da sua plataforma**: aceda ao separador `Contas de e-mail`{.action}. À direita da conta a eliminar, clique no botão `...`{.action} e, a seguir, `Renovar esta conta`{.action} ou `Reinicializar`{.action}.
```

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]

File: pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md (hunk 5 above)
Comment (BrenKt) [reply pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md:39]: this part is like FR version, nothing is missing

**Review by OVH-Team-Guides — APPROVED** [review]

**Pull request merged**